avogadro/qtplugins/surfaces/surfaces.cpp

```diff
@@ -190,8 +192,53 @@ void Surfaces::calculateSurface()
 
 void Surfaces::calculateEDT()
 {
-  // pass the molecule to the EDT, plus the surface type
-  // get back a Cube object in m_cube
+  /*double probeRadius = 0.0;
+  switch (m_dialog->surfaceType()) {
+    case VanDerWaals:
+    case SolventAccessible:
+    case SolventExcluded:
+  }*/
+
+  // cache Van der Waals radii
+  std::vector<double> radii(m_molecule->atomCount());
+  double max_radius = probeRadius;
+  for (size_t i = 0; i < radii.size(); i++) {
+    radii[i] = Core::Elements::radiusVDW(m_molecule->atomicNumber(i));
+    if (radii[i] <= 0.0)
```

Comment:
I don't think this can ever happen. They're all above 0.0. Radii are in `core/elementdata.h`

Reply:
Done!